src/main/resources/standardTestCases/globbingindex/globbingindex.wdl

```diff
@@ -0,0 +1,60 @@
+task glob_task {
+    command {
+        echo "staticFile" > staticFile.txt
+        echo "staticArray" > staticArray.txt
+        echo "globFile" > globFile.txt
+        echo "globArray" > globArray.txt
+    }
+
+    output {
+        Array[File] staticFiles = ["staticFile.txt"]
+        Array[File] staticRef = staticFiles
+        File staticFile = staticFiles[0]
+        Array[File] staticArray = ["staticArray.txt"]
+        File globFile = glob("globFile.txt")[0]
+        Array[File] globArray = glob("globArray.txt")
```

Comment:
There are a number of expressions that "look good to me?"™ but either don't parse `WdlExpression.fromString(<expr>)`, or return an AST that is effectively empty. I'll add them to a new ticket.

For example:
- `myArray[read_int("intFile.txt")]`
- `[0, 1, 2][0]`
- `{ "a": 1 }[a]`